V2/time_tracker/time_entries/_application/_time_entries/_create_time_entry.py

```diff
@@ -0,0 +1,65 @@
+import dataclasses
+import json
+import typing
+
+import azure.functions as func
+
+from ... import _domain
+from ... import _infrastructure
+from time_tracker._infrastructure import DB
+
+_DATABASE = DB()
```

Comment:
Please, let'd not instantiate the DB in the module, let's do it inside the function. Instantiating the object in the module will make testing harder